server/src/controllers/appointments.controller.js

```diff
@@ -45,7 +51,8 @@ export async function newAppointment(req, res, next) {
         preferredEarlyTime: earlyTimeHour,
         preferredLateTime: lateTimeHour,
       },
-      location: { ...coords, address },
+      location: geocodeData.results[0].geometry.location,
```

Comment:
`location` is a subdocument so the geocoded data and the address need to be spread into a new object

Reply:
Do you mean like this?

location: {
        address: address,
        lat: geocodeData.results[0].geometry.location.lat,
        long: geocodeData.results[0].geometry.location.lat,
      },

Reply:
That should work